pkgs/development/haskell-modules/configuration-ghc-7.8.x.nix

```diff
@@ -75,6 +75,9 @@ self: super: {
   ghc-exactprint = dontDistribute super.ghc-exactprint;
   ghc-typelits-natnormalise = dontDistribute super.ghc-typelits-natnormalise;
 
+  # Needs directory >= 1.2.2.0.
+  idris = dontDistribute super.idris;
```

Comment:
Minor nit-pick: `dontDistribute` means that Hydra should not build this package. Hydra doesn't build the ghc784 package set anyway, though, so while it doesn't hurt to specify this here, it's probably unnecessary.

Also, you should probably use `markBroken` here because the build is known to fail, so there's no point even trying to instantiate this derivation.